src/modules/airline/index.ts

```diff
@@ -0,0 +1,116 @@
+import type { Faker } from '../..';
+
+export enum Aircraft {
+  Narrowbody = 'narrowbody',
+  Regional = 'regional',
+  Widebody = 'widebody',
+}
+
+export type AircraftType = `${Aircraft}`;
+
+const numerics = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9'];
+const visuallySimilarCharacters = ['0', 'O', '1', 'I', 'L'];
+const aircraftTypeMaxRows = {
+  regional: 20,
+  narrowbody: 35,
+  widebody: 60,
+};
+const aircraftTypeSeats = {
+  regional: ['A', 'B', 'C', 'D'],
+  narrowbody: ['A', 'B', 'C', 'D', 'E', 'F'],
+  widebody: ['A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'J', 'K'],
+};
+
+/**
+ * Module to generate airline related data.
+ */
+export class AirlineModule {
+  constructor(private readonly faker: Faker) {
+    // Bind `this` so namespaced is working correctly
+    for (const name of Object.getOwnPropertyNames(AirlineModule.prototype)) {
+      if (name === 'constructor' || typeof this[name] !== 'function') {
+        continue;
+      }
+      this[name] = this[name].bind(this);
+    }
+  }
+
+  /**
+   * Generates a random IATA airport code.
+   *
+   * @example
+   * faker.airline.airportCode() // 'JFK'
+   *
+   * @since 8.0.0
+   */
+  airportCode(): string {
+    return this.faker.helpers.arrayElement(
+      this.faker.definitions.airline.airport
+    );
+  }
+
+  /**
+   * Generates a random record locator.
```

Comment:
Maybe add the link to the top (our other links are also like this). Also maybe add a less technical term for it so it is easier to search on the website.

```suggestion
   * Generates a random [record locator](https://en.wikipedia.org/wiki/Record_locator) (reservation code).
```

What do you think?

Reply:
Good suggestion. I added a little more explanatory text as well.